flusher.go

```diff
@@ -126,6 +129,28 @@ func (s *Server) Flush(ctx context.Context) {
 	}()
 }
 
+type timeseriesSummary struct {
+	tags  []string
+	count int64
+}
+
+func (s *Server) tallyTimeseries() timeseriesSummary {
+	tags := []string{fmt.Sprintf("global_veneur:%t", !s.IsLocal())}
+	count := int64(0)
```

Comment:
you can just do `var count int64` (it defaults to the zero value for the type, which in this case is zero)